addons/im_livechat/static/src/models/mobile_messaging_navbar/mobile_messaging_navbar.js

```diff
@@ -0,0 +1,25 @@
+/** @odoo-module **/
+
+import { patchRecordMethods } from '@mail/model/model_core';
+// ensure that the model definition is loaded before the patch
+import '@mail/models/mobile_messaging_navbar_view/mobile_messaging_navbar_view';
+
+patchRecordMethods('MobileMessagingNavbarView', {
+    /**
+     * @override
+     */
+    _computeTabs() {
+        const res = this._super();
+        const livechat = this.messaging.models['mail.thread'].find(
+            thread => thread.channel_type === 'livechat',
+        );
```

Comment:
This is the kind of badly scaling compute that we need to avoid though. I guess it is fine for now, but I'll need to adapt it when forward-porting my PR.

Reply:
There is an immediate solution for such an easy case though:

Move the compute to thread, where thread is having a "discussAsLivechat" kind of relation, and fill it only if the type is right.
For the compute here, read the length on the inverse relation (on discuss).